examples/stocks/lib/stock_data.dart

```diff
@@ -76,7 +76,7 @@ class StockDataFetcher {
       String json = response.body;
       if (json == null) {
         print("Failed to load stock data chunk ${_nextChunk - 1}");
-        return;
+        return null;
```

Comment:
This shouldn't be needed. `return` without a value should just return null, no? We rely on this all over the place.